multiuser/keycloak/che-multiuser-keycloak-user-remover/src/test/java/org/eclipse/che/multiuser/keycloak/server/KeycloakUserRemoverTest.java

```diff
@@ -63,7 +64,7 @@ public void shouldCatchFileNotFoundException() throws Exception {
     try {
       keycloakUserRemover.removeUserFromKeycloak("123");
     } catch (Exception e) {
-      fail("Should not have thrown an exception.");
+      fail();
```

Comment:
you don't need to fail at all here, as well as try-catch block. The whole test will fail anyway if this block throw and error that is not declared as expected